tests/testthat/test-get_delay_estimate.R

```diff
@@ -122,3 +175,75 @@ test_that("get_delay_estimate handles partially complete reporting triangles", {
     n = 5
   ))
 })
+
+test_that("get_delay_estimate works with weekly reporting of daily data", {
+  sim_delay_pmf <- c(0.1, 0.2, 0.3, 0.1, 0.1, 0.1, 0.1)
+
+  counts <- c(100, 20, 30, 40, 50, 60, 70)
+
+  complete_triangle <- lapply(counts, function(x) x * sim_delay_pmf)
+  complete_triangle <- do.call(rbind, complete_triangle)
+
+  reporting_triangle <- matrix(
```

Comment:
Isn't this a 7x7? Wouldn't weekly reporting of daily data be a 7 x 1, 14 x 2 etc? Also think we should test out the short triangle which would have a triangle of  n/7 x n ...